### DISCUSSION TITLE
SVG runtime load library

### DISCUSSION THREAD

**Discussion by marchbold** [airsdk/Adobe-Runtime-Support#697] [Q&A]
Does anyone know of a good SVG loading library? I need to dynamically load in an SVG at runtime and display it to the user. 

I've found two older libs that seem to cover some use cases:

- https://github.com/lucaslorentz/AS3SVGRenderer
- https://code.google.com/archive/p/svgweb/

but I was wondering if anyone know's of any maintained libs?

**Comment by MESepehr** [comment]
only I saw this example for AS3SVGRenderer
https://stackoverflow.com/questions/10866636/actionscript-3-import-svg-file

**Reply by marchbold** [reply]
Cheers, that looks like a reference to the first lib I listed, and they seem to have the same issue I encountered with it, i.e. not correctly detected the size of the svg and hence it doesn't resize to fit correctly. 

The second one seems to work better, so far.
